actionview/lib/action_view/helpers/text_helper.rb

```diff
@@ -103,7 +103,9 @@ def truncate(text, options = {}, &block)
       # Highlights one or more +phrases+ everywhere in +text+ by inserting it into
       # a <tt>:highlighter</tt> string. The highlighter can be specialized by passing <tt>:highlighter</tt>
       # as a single-quoted string with <tt>\1</tt> where the phrase is to be inserted (defaults to
-      # '<mark>\1</mark>') or passing a block that receives each matched term.
+      # '<mark>\1</mark>') or passing a block that receives each matched term. By default the text
+      # will be sanitized. If you agree with security risk you can turn it off. For this you need
+      # set <tt>:sanitize</tt> option to +false+.
```

Comment:
Let's change it a bit more: "By default +text+ is sanitized to prevent possible XSS attacks. If the input is trustworthy, passing false for <tt>:sanitize</tt> will turn sanitizing off."